virtualizarr/writers/icechunk.py

```diff
@@ -144,8 +144,7 @@ def write_virtual_variable_to_icechunk(
     # TODO it would be nice if we could assign directly to the .attrs property
     for k, v in var.attrs.items():
         arr.attrs[k] = encode_zarr_attr_value(v)
-    arr.attrs["_ARRAY_DIMENSIONS"] = encode_zarr_attr_value(var.dims)
```

Comment:
It's not necessary to write these because we already have `dimension_names` specified above in

```python
arr = group.require_array(
        name=name,
        shape=zarray.shape,
        chunk_shape=zarray.chunks,
        dtype=encode_dtype(zarray.dtype),
        codecs=zarray._v3_codec_pipeline(),
        dimension_names=var.dims,
        fill_value=zarray.fill_value,
        # TODO fill_value?
    )
```